### PR TITLE
[tests-only] Use .drone.star in CI

### PR DESCRIPTION
**Pull request by saw-jan**
Changes:
- renamed `.drone.starlark` to `.drone.star`
- deleted `.drone.yml`
- removed the special makefile target that was for generating `.drone.yml`

Closes https://github.com/owncloud/user_ldap/issues/632

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: drone CI is found at https://drone.owncloud.com/owncloud/user_ldap

I just changed https://drone.owncloud.com/owncloud/user_ldap/settings so that it uses `.drone.star`

@saw-jan please push something for this PR so that it triggers drone again.

File: .drone.star [hunk 1]
```diff
@@ -917,7 +917,7 @@ config = {
 						'bash ./apps/user_ldap/tests/acceptance/setConfig.sh',
 						'php occ ldap:show-config',
 						'php occ ldap:test-config "LDAPTestId"',
-						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:sync "OCA\\\User_LDAP\\\User_Proxy" -m remove',
```
Comment (saw-jan) [inline .drone.star:920]: `\\` didn't work here so tried with `\\\` and surprisingly it worked.

Comment (phil-davis) [reply .drone.star:920]: I guess there are 2 levels of escaping in the drone starlark extension on the server and somewhere in the internal yml and processing commands for execution. Special magic!

**Review by phil-davis — APPROVED** [review]
Well found - :rocket:

**Pull request merged**